### ISSUE TITLE
classpath collisions with lein-collisions?

### ISSUE THREAD

**Issue by benedekfazekas** [webnf/lein-collisions#1]
i can't seem to use `lein-collisions` and `lein-cljs-lambda` plugins together. I get

```
Caused by: clojure.lang.Compiler$CompilerException: java.io.FileNotFoundException: Could not locate clojure/data/priority_map__init.class or clojure/data/priority_map.clj on classpath: , compiling:(clojure/core/cache.clj:1:1)
 at clojure.lang.Compiler.load (Compiler.java:7142)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.loadResourceScript (RT.java:361)
    clojure.lang.RT.load (RT.java:440)
    clojure.lang.RT.load (RT.java:411)
```

perhaps `lein-collisions` is colliding with `lein-cljs-lambda`? if i remove `lein-collisions` from my plugin list problem is fixed.

**Comment by bendlas** [comment]
Hm, I can reproduce this, but in a test project with

``` clj
:dependencies
[[io.nervous/cljs-lambda "0.1.1"]
 [io.nervous/lein-cljs-lambda "0.2.0"]
 [lein-collisions "0.1.2"]]
```

there are no apparent file collisions.

Investigating ...

**Comment by bendlas** [comment]
As far as I could discern, the problem is with some of the classloader magic, that leiningen uses to set up its plugin runtime. The bug is a really shifty beast, triggered by `(require 'leiningen.new.template)`, but `(require 'clojure.data.priority-map 'leiningen.new.template)` works.

So I just released 0.1.3, which repackages the library code into the plugin. This seems to also fix this issue.

**Comment by benedekfazekas** [comment]
nice one! will give it a spin
